sdcm/tester.py

```diff
@@ -574,7 +574,7 @@ def get_cluster_gce(self, loader_info, db_info, monitor_info):
                                            **common_params)
 
         loader_additional_disks = {'pd-ssd': self.params.get('gce_pd_ssd_disk_size_loader', default=0)}
-        self.loaders = LoaderSetGCE(gce_image=self.params.get('gce_image'),
+        self.loaders = LoaderSetGCE(gce_image=self.params.get('gce_image_loader'),
```

Comment:
Should it be gce_image=self.params.get('gce_image_loader', self.params.get('gce_image')) ?

Reply:
Also there is another place in get_cluster_k8s_gce_minikube, where we initialize LoaderSetGCE, shouldn't we have it fixed it too ?